### DISCUSSION TITLE
[Esame 6 Settembre 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#372] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2a184793-79d8-4b4b-9fad-6a199322cc02)

**Comment by Frisbon** [comment]
![image_2024-01-30_11-29-12](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/e55909b4-b413-4e8e-8f55-c62a38b9eccc)

Dato che $\forall a \ m_g(\lambda) \not = m_a(\lambda)$ affermo che la matrice non é diagonalizzabile in nessun modo

**Reply by Traster-uni** [reply]
Ciao, mi sembra ci sia una svista nel calcolo del determinante, ti sei perso un quadrato, dovrebbe essere 
` 2a (1-lambda)^3 + 2(a - alambda)`

**Reply by Frisbon** [reply]
Ho rifatto l'esercizio perché era totalmente errato, mi viene che non é diagonalizzabile al variare di a.
